src/test/java/core/basesyntax/service/InputDataServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import org.junit.Test;
+
+public class InputDataServiceImplTest {
+    @Test
+    public void stringToFruitTransactionConverter_validInputTextFormat_ok() {
+        InputDataService inputDataService = new InputDataServiceImpl();
```

Comment:
Make tested service class-level field - you use it in all methods.

**Same for other test classes.**